src/main/twirl/com/lightbend/lagom/docs/sideNav.scala.html

```diff
@@ -4,16 +4,16 @@
 
 <h3>Documentation</h3>
 <ul>
-    <li><a href="@context.path/get-started.html">Try Lagom</a></li>
-    <li><a href="@context.path/documentation/">Full documentation</a></li>
-    <li><a href="@context.path/faq.html">FAQ</a></li>
-    <li><a href="@context.path/blog">Blog</a></li>
+    <li><a href="@context.path/documentation">By release</a></li>
+    <li><a href="@context.path/get-started.html">Get started</a></li>
```

Comment:
Why change "Try Lagom" back to "Get started"? The link in the header still says "Try Lagom".

Reply:
Because having two Try Lagom items on the same menu looks like a mistake. I guess an alternate fix would be to change the heading of the menu, but I didn't look into it. And, if someone clicks on Get Started and lands on the Try Lagom page, they shouldn't be confused.

Reply:
I looked and it must be that the Try Lagom heading at the top is coming from the implicit context? Anyway, it doesn't matter which page you are on, Try Lagom is always at the top of the menu on a mobile device (at least on the emulators), and it links to the Try Lagom page, so I just removed the Get Started item from the side menu (which turns into the mobile menu). I think this is a better solution.

Reply:
The top-level Try Lagom heading is hardcoded into the mobile menu before bringing in `sideNav`

https://github.com/lagom/lagom.github.io/blob/a56fd98ac8fce01e01f418400dbd5dc52f3da485/src/main/twirl/com/lightbend/lagom/docs/main.scala.html#L24-L30

Your solution seems like a good one.